streams/src/test/java/org/apache/kafka/streams/processor/internals/StreamsPartitionAssignorTest.java

```diff
@@ -1402,9 +1404,13 @@ public void shouldTriggerImmediateRebalanceOnTasksRevoked() {
         final Map<String, Assignment> assignment = partitionAssignor.assign(metadata, new GroupSubscription(subscriptions)).groupAssignment();
 
         // Verify at least one partition was revoked
-        assertThat(assignment.get(CONSUMER_1).partitions(), not(allTasks));
+        assertThat(assignment.get(CONSUMER_1).partitions(), not(allPartitions));
```

Comment:
I found it was always true because we are comparing a list of tasks with a list of partitions ;)

Reply:
Whoops

Reply:
Oh, boy...